recipe/meta.yaml

```diff
@@ -42,12 +42,10 @@ requirements:
   host:
     - mpifx * mpi_{{ mpi }}_*  # [mpi != "nompi"]
     - libblas
-    - hdf5 >=1.10.4
+    - hdf5
```

Comment:
I don't have much experience with packaging yet, could you explain to me why omitting the lower bound is advantageous in this case?

Reply:
The build time and runtime versions of C packages are related to one and other.

HDF5 dictates that at run time, the same version, down to the patch, must be used https://github.com/conda-forge/hdf5-feedstock/blob/main/recipe/meta.yaml#L66

At conda-forge, we have a bot that will rebuild recipes https://conda-forge.org/status/ when a new version of a package is detected.

However, the `>` pinning doesn't let the bot do its job.

In this case, HDF5 is set to 1.12.2
https://github.com/conda-forge/fortnet-feedstock/pull/2/files#diff-55deb6198939adcb12c14739035d797303c85c7e1f1bd16fea1bb0a202e8d6a6R18

Reply:
I see, thanks for explaining it.